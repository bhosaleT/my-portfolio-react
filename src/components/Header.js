import React from "react";
import Selector from './Selector';

class Header extends React.Component {
  render() {
    return (
      <div>
        <div className="header">
          <img
            className="profile-image"
            src="https://static.tvtropes.org/pmwiki/pub/images/eric-cartman_380.jpg"
            alt="Tejas-Bhosale"
          />
          <div>
            <h2 className="name">Tejas Bhosale</h2>
            <p className="title">Full Stack Developer || Game Developer</p>
          </div>
          <Selector />
        </div>
        <hr className="hr"/>
      </div>
    );
  }
}
export default Header;
