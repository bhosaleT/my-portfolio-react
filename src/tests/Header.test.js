import ReactShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import Header from '../components/Header';

test('The skills component should render without crashing', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<Header />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
})