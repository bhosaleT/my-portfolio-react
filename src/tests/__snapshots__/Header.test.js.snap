// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The skills component should render without crashing 1`] = `
<div>
  <div
    className="header"
  >
    <div
      className="info"
    >
      <h2
        className="name"
      >
        Tejas Bhosale
      </h2>
      <p
        className="title"
      >
        Full Stack Developer 
        <span
          className="divider"
        >
           || 
        </span>
         Game Enthusiast
      </p>
    </div>
  </div>
</div>
`;
